E/Exercism/Compat.toml

```diff
@@ -1,3 +1,8 @@
 [0]
-JSON = "0.19-0"
 julia = "0.7-1"
+
+["0-0.1.2"]
+JSON = "0.19-0"
+
+["0.1.3-0"]
+JSON = "0.19"
```

Comment:
Doesn't work with JSON 0.20? I am guessing it does. You probably want
```toml
[compat]
JSON = "0.19, 0.20"
```
in your project file.

Reply:
It does work with 0.20 indeed. I misread how `^` works on 0.x versions. Thanks for your patience :D